### ISSUE TITLE
in-line code brackets `[]` turned into `<>`

### ISSUE THREAD

**Issue by dingyifei** [mondeja/mdpo#84]
After po2md,  \`[]\` turned into \`<>\`.
Input
```
msgid ""
"This document describes the method for calibrating the x, y, and z "
"offsets of an \"automatic z probe\" in Klipper. This is useful for "
"users that have a `[probe]` or `[bltouch]` section in their config "
"file."
msgstr ""
```

Output
```
This document describes the method for calibrating the x, y, and z offsets of an
"automatic z probe" in Klipper. This is useful for users that have a
`<probe>` or `<bltouch>` section in their config file.
```

**Comment by mondeja** [comment]
This should have been fixed in [v0.3.18](https://pypi.org/project/mdpo/0.3.18/). Could you confirm it?

**Comment by dingyifei** [comment]
It's fixed :P
